src/components/PageSections/Menu/FullSiteMenu.js

```diff
@@ -13,43 +13,66 @@ const FullSiteMenu = props => {
   const isMobile = useMobileQuery();
 
   return (
-    <div
-      ref={props.refnode}
-      className={
-        'coa-FullSiteMenu ' +
-        (props.isTopMenuActive ? 'coa-FullSiteMenu--active' : '')
-      }
-    >
-
-      <div className="wrapper">
-        <ThemesNav
-          handleOnClick={props.handleFullSiteMenuOpen}
-          isTopMenuActive={props.isTopMenuActive}
-          handleFullSiteMenuClose={props.handleFullSiteMenuClose}
-          navigation={props.navigation}
-          slug={props.slug}
-        />
-      </div>
+    <div>
 
-      {!isMobile &&
-        <section className="coa-FullSiteMenu__container">
-          <div className="wrapper container-fluid">
-            <div className="coa-MessageWIP">
-              <WorkInProgressBanner />
-            </div>
-          </div>
-        </section>
-      }
+      <div
+        ref={props.refnode}
+        className={
+          'coa-FullSiteMenu ' +
+          (props.isTopMenuActive ? 'coa-FullSiteMenu--active' : '')
+        }
+      >
 
-      <section className="coa-FullSiteMenu__subNav">
-        <div className="wrapper container-fluid">
-          <ThemesTopicsMenu
-            menu={props.navigation}
-            handleFullSiteMenuItem={props.handleFullSiteMenuItem}
+        <div className="wrapper">
+          <ThemesNav
+            handleOnClick={props.handleFullSiteMenuOpen}
+            isTopMenuActive={props.isTopMenuActive}
+            handleFullSiteMenuClose={props.handleFullSiteMenuClose}
+            navigation={props.navigation}
             slug={props.slug}
           />
         </div>
-      </section>
+
+          <div className={
+              (props.isTopMenuActive ? 'coa-FullSiteMenu_dropdown' : '')
+            }
+          >
+
+            {!isMobile &&
+              <section className="coa-FullSiteMenu__container">
+                <div className="wrapper container-fluid">
+                  <div className="coa-MessageWIP">
+                    <WorkInProgressBanner />
+                  </div>
+                </div>
+              </section>
+            }
+
+            <section className="coa-FullSiteMenu__subNav">
+              <div className="wrapper container-fluid">
+                <ThemesTopicsMenu
+                  menu={props.navigation}
+                  handleFullSiteMenuItem={props.handleFullSiteMenuItem}
+                  slug={props.slug}
+                />
+              </div>
+            </section>
+
+          </div>
+
+      </div>
+
+      { !isMobile && props.isTopMenuActive &&
+
+        /*
+          This div acts as a gap preserver for when the dropdown menu is active,
+          which changes the mneu position to absolute, thus the static gap is lost.
+          Here we recreate that.
+        */
+
+        <div className="coa-FullSiteMenu__activeGapFill"></div>
+
+      }
```

Comment:
I updated this div to have it's own class rather than inline styles + added notes for clarification. There's probably a better way to handle this. But, as far as I can tell with some messing around, some serious site wide structural changes would need to be made.

Reply:
Ok cool, that sounds like a good solution.